### ISSUE TITLE
Extreme lag when starting/updating

### ISSUE THREAD

**Issue by worldendDominator** [Lailloken/Lailloken-UI#483]
Whenever the script starts up or updates, my cursor movement starts massively lagging, I'm talking moving every half second or so. It takes several seconds to stop.

**Comment by Lailloken** [comment]
I assume this started happening with the latest version. I changed how the client.txt is parsed on startup if the "current char" is specified in the general tab of the settings.

Do you have a "current char" set, and how large is your client.txt?

**Comment by worldendDominator** [comment]
No, I've been noticing this for several leagues, I just never got around to report it.

My client.txt is 120 megabytes big, I guess that's the reason.

**Comment by Lailloken** [comment]
Hmm, okay. You could try renaming the client.txt and let the game create a new one (or simply delete it if you don't need it for other log-related apps).

Let me know if that does anything for startup-times.
